app/models/game.js

```diff
@@ -0,0 +1,7 @@
+import DS from 'ember-data';
+
+export default DS.Model.extend({
+  createdDate: DS.attr('date'),
+  gameParticipants: DS.hasMany('gameParticipant', {embedded: true}),
+  isStarted: DS.attr('boolean', {defaultValue: false})
```

Comment:
I was thinking about this, we may want more than one status like 'preparing', 'in-play', 'ended' and maybe others?